app/Models/User.php

```diff
@@ -78,7 +78,7 @@ public function scopeFilterUser($query, QueryFilter $filters)
 
     public function getShiftByDate($date)
     {
-        $work_schedule = $this->work_schedules()
+        $work_schedule = $this->workSchedules()
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Models/User.php</br>Variable "work_schedule" is not in valid camel caps format